parsers/jscript.c

```diff
@@ -1741,10 +1775,12 @@ static bool parseMethods (tokenInfo *const token, const tokenInfo *const class,
 					if (isKeyword (token, KEYWORD_async))
 						readToken (token);
 				}
+
+				vString * signature = NULL;
```

Comment:
You didn't assign a vString here, you assigned at ... *1*

Reply:
*2*, here. So we can make the code simplified like:

```
vString * signature = vStringNew();
{
}
vStringDelete(signature);
```
This function is already enough long. So I would like to avoid introducing extra complexity.

I would like to call the approach I suggested here *1New1Delete* in this comment.